### PR TITLE
Add memgraph/v1.2.0 pages

### PR DESCRIPTION
**Pull request by gitbuda**

### CODE REVIEW

File: products/memgraph/v1.2.0/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  * [About Memgraph](#about-memgraph)
+  * [Quick Start](quick-start.md)
+  * [Tutorials](tutorials/tutorials-overview.md)
+    * [Analyzing TED Talks](tutorials/analyzing-TED-talks.md)
+    * [Graphing the Premier League](tutorials/graphing-the-premier-league.md)
+    * [Exploring the European Road Network](tutorials/exploring-the-european-road-network.md)
+    * [Backpacking Through Europe](tutorials/backpacking-through-europe.md)
+    * [Movie Recommendation System](tutorials/movie-recommendation.md)
+    * [Marvel Comic Universe Social Network](tutorials/marvel-universe.md)
+    * [Football Transfers](football-transfers.md)
```
Comment (antonio2368) [inline products/memgraph/v1.2.0/README.md:34]: Missing music tutorial

File: products/memgraph/v1.2.0/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+To verify that Memgraph is running, run the following command:
+
+```plaintext
+journalctl --unit memgraph
+```
+
+If successful, the you should receive an output similar to the following:
+
+```plaintext
+Nov 23 13:40:13 hostname memgraph[14654]: Starting 8 BoltS workers
```
Comment (antonio2368) [inline products/memgraph/v1.2.0/quick-start.md:104]: Missing the connection information output but not critical

File: products/memgraph/v1.2.0/README.md (hunk 1 above)
Comment (gitbuda) [reply products/memgraph/v1.2.0/README.md:31]: done

File: products/memgraph/v1.2.0/quick-start.md (hunk 2 above)
Comment (gitbuda) [reply products/memgraph/v1.2.0/quick-start.md:105]: done

**Review by antonio2368 — APPROVED** [review]

**Pull request merged**